ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -932,6 +932,23 @@ extension BrowserViewController: WKNavigationDelegate {
 
     // The challenge may come from a background tab, so ensure it's the one visible.
     tabManager.selectTab(tab)
+    tab.isDisplayingBasicAuthPrompt = true
+    defer { tab.isDisplayingBasicAuthPrompt = false }
+
+    let isHidden = webView.isHidden
+    defer { webView.isHidden = isHidden }
+
+    // Manually trigger a `url` change notification
+    if host != tab.url?.host {
```

Comment:
So we are not hiding the webview If the hosts are same and using as it is correct ?